### PR TITLE
read-me updated

### PR DESCRIPTION
**Pull request by humulla**
### **Link to JIRA ticket:**
https://hackney.atlassian.net/jira/software/projects/PA/boards/35?selectedIssue=PA-340

**_What changes have we introduced:_**
I have made changes to the readMe file so it links to Tenancy API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # LBH Tenancy Information API
 
 Tenancy Information API returns information about tenancies held
```
Comment (Renny4Real) [inline README.md]: Could we make this clearer too? Maybe use what's in swagger?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+The API has 2 endpoints one for retrieving all information aboout tenancies and one for a specific ID
```
Comment (Renny4Real) [inline README.md]:
```suggestion
The API has 2 endpoints, one for retrieving all tenancy records and one for retrieving a single record for a specific ID
```

**Review by Renny4Real — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (humulla) [reply README.md]: Swagger one is not great:
API for accessing tenancy information

Comment (Renny4Real) [inline README.md]:
```suggestion
Tenancy Information API returns information about tenancies
```

**Review by Renny4Real — APPROVED** [review]

**Pull request merged**